opensourceleg/actuators/moteus.py

```diff
@@ -31,56 +38,98 @@
     ActuatorIsNoneException,
     ControlModeException,
 )
-from opensourceleg.logging import LOGGER
 from opensourceleg.logging.decorators import (
     deprecated,
     deprecated_with_routing,
     deprecated_with_suggestion,
 )
 
+from opensourceleg.math import ThermalModel
+
+from opensourceleg.safety import ThermalLimitException
+
 from opensourceleg.logging.logger import LOGGER
 
 
-DEFAULT_POSITION_GAINS = ControlGains(kp=0, ki=0, kd=0, k=0, b=0, ff=0)
+# Default gains to be tuned
+DEFAULT_POSITION_GAINS = ControlGains(kp=4.00, ki=1.00, kd=0.05, k=0, b=0, ff=0)
+
+DEFAULT_VELOCITY_GAINS = ControlGains(kp=4.00, ki=1.00, kd=0.05, k=0, b=0, ff=0)
 
 DEFAULT_CURRENT_GAINS = ControlGains(kp=0, ki=0, kd=0, k=0, b=0, ff=0)
 
 DEFAULT_IMPEDANCE_GAINS = ControlGains(kp=0, ki=0, kd=0, k=0, b=0, ff=0)
 
+class MoteusQueryResolution:
 
-class MoteusVoltageMode(ControlModeBase):
+    mode = mp.INT8
+    position = mp.F32
+    velocity = mp.F32
+    torque = mp.F32
+    
+    voltage = mp.INT8
+    temperature = mp.INT32
+    fault = mp.INT8
+
+    # Kept for Controller init reference
+    q_current = mp.INT32
+    d_current = mp.IGNORE
+    abs_position = mp.IGNORE
+    power = mp.IGNORE
+    motor_temperature = mp.IGNORE
+    trajectory_complete = mp.IGNORE
+    rezero_state = mp.IGNORE
+    home_state = mp.IGNORE
+
+    aux1_gpio = mp.IGNORE
+    aux2_gpio = mp.IGNORE
+
+    _extra = {
+        # MoteusRegister.POSITION_KP: mp.F32, 
+        # MoteusRegister.POSITION_KI: mp.F32, 
+        # MoteusRegister.POSITION_KD: mp.F32, 
+        MoteusRegister.COMMAND_POSITION: mp.F32,
+        MoteusRegister.COMMAND_VELOCITY: mp.F32,
+        MoteusRegister.COMMAND_FEEDFORWARD_TORQUE: mp.F32, 
+        MoteusRegister.COMMAND_Q_CURRENT: mp.F32, 
+    }
+
+class MoteusStopMode(ControlModeBase):
```

Comment:
Stop mode is not a control mode and maybe we can achieve this functionality with a function instead?